test/admin/dev.test.js

```diff
@@ -97,3 +97,55 @@ describe('Dev V2', () => {
     ]);
   });
 });
+
+const FUNC_NAME = 'myFunction';
+const FUNC_SOURCE = 'exports = function(arg1, arg2){ return {sum: arg1 + arg2, userId: context.user.id}}';
+const myFunction = () => ({ name: FUNC_NAME, source: FUNC_SOURCE });
```

Comment:
nit: doesn't look this needs to be a function - you could just use an object literal